src/group/mls_group/test_mls_group.rs

```diff
@@ -125,3 +127,183 @@ fn test_failed_groupinfo_decryption() {
         }
     }
 }
+
+#[test]
+/// Test what happens if the KEM ciphertext for the receiver in the UpdatePath is broken.
+fn test_update_path() {
+    for ciphersuite in Config::supported_ciphersuites() {
+        // Basic group setup.
```

Comment:
Don't we have a test framework to do all this setup stuff?

Reply:
We do, but it lives in `./tests/utils` and is more suited for integration testing. The test I'm doing is more of a unit test and requires a deep-dive into various data structures that I'm not sure I can easily access from outside.